pkg/sql/crdb_internal.go

```diff
@@ -9053,7 +9053,7 @@ CREATE TABLE crdb_internal.cluster_replication_node_streams (
 
 			if err := addRow(
 				tree.NewDInt(tree.DInt(s.StreamID)),
-				tree.NewDInt(tree.DInt(s.Spec.ConsumerID)),
+				tree.NewDString(fmt.Sprintf("%d[%d]", s.Spec.ConsumerNode, s.Spec.ConsumerProc)),
```

Comment:
I assume the goal of keeping this as one field is to keep the total table width down?

Reply:
Indeed. I have a few more new columns coming and want to try to keep it one line . 

Conceptually this column was originally added to uniquely identify the consumer and did so back when each node ran one proc. When each node runs more than one proc it can still uniquely identify the consumer without needing more columns this way.